_rules/SC1-2-1-media-alternative-video.md

```diff
@@ -42,7 +42,7 @@ There are no major accessibility support issues known for this rule.
 
 ## Background
 
-- 
+- https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html
```

Comment:
Add the title of the page as the text [Tile of page](link to page)

Reply:
Fixed. Thanks